WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Success card/MigrationSuccessCardView.swift

```diff
@@ -73,13 +73,20 @@ class MigrationSuccessCardView: UIView {
     }
 }
 
-// TODO: This extension is temporary, and should be replaced by the actual condition to check, and placed in the proper location
+// Ideally, this logic should be handled elsewhere. But since the whole migration feature is temporary
+// Perhaps that's not worth the trouble.
+//
+// TODO: Remove `shouldShowMigrationSuccessCard` when the migration feature is no longer needed
 extension MigrationSuccessCardView {
-    @objc
-    static var shouldShowMigrationSuccessCard: Bool {
 
-        AppConfiguration.isJetpack && showCard
+    @objc static var shouldShowMigrationSuccessCard: Bool {
+        #if JETPACK
```

Comment:
`AppConfiguration.isJetpack` wouldn't work because `MigrationAppDetection` file is Jetpack-only, whereas `MigrationSuccessCardView` is added to both `WordPress` and `Jetpack` targets. So if we use the runtime check, the WordPress target won't build.

An alternative to compiler directives, is to do the following:
```swift
// MigrationSuccessCardView+Jetpack.swift added to Jetpack target
extension MigrationSuccessCardView {

    @objc static var shouldShowMigrationSuccessCard: Bool {
        let isFeatureFlagEnabled = FeatureFlag.contentMigration.enabled
        let isWordPressInstalled = MigrationAppDetection.getWordPressInstallationState().isWordPressInstalled
        let isMigrationCompleted = UserPersistentStoreFactory.instance().isJPContentImportComplete
        return isFeatureFlagEnabled && isWordPressInstalled && isMigrationCompleted
    }
}

// MigrationSuccessCardView+WordPress.swift added to WordPress target
extension MigrationSuccessCardView {

    @objc static var shouldShowMigrationSuccessCard: Bool {
         return false
    }
}
```

Both solutions aren't perfect, but I slightly prefer the compiler directives over adding 2 files `MigrationSuccessCardView+Jetpack.swift` and `MigrationSuccessCardView+WordPress.swift`

Reply:
Or we can just add it to the wp target as well. This isn't perfect either, but since wp itself will undergo changes in the near future and this is temporary, maybe we can do this in order to simplify the code?

Reply:
Sure thing. Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/19693/commits/7c1b1b7496d23eb475248c5d4c87c4e8f71d63ac